packages/react-juce/src/lib/Backend.ts

```diff
@@ -99,6 +100,15 @@ export class ViewInstance {
     // of a css prop, first convert it to kebab-case
     propKey = cssPropsMap[propKey] || propKey;
 
+    // convert provided color string to alpha-hex code for JUCE
+    const colors = new Colors();
+    if (colors.isColorProperty(propKey)) {
+      value = colors.colorStringToAlphaHex(value);
+      if (value.startsWith("linear-gradient")) {
+        value = colors.convertLinearGradientStringToNativeObject(value);
```

Comment:
One thought here: might we be interested in holding on to the `value` given by the user when we write to `this._props`? We only need this computed value when we speak to native, I could imagine it making sense to otherwise continue to deal in "non-computed" property values at the javascript level.

In which case, I would just say let's do something like `let nativeValue = colors.convertLinearGradientStringToNativeObject(value);` and then pass `nativeValue` below in the NativeMethods call, and pass `value` to the `Object.assign()` going onto `this._props`.

Reply:
roger that. can see reusing `this._props` being useful for say presenting the user with an error when passing in an invalid prop value. Sure there's lot of other useful reasons we might want to access props besides just during rendering too.